demos/frontend-react/README.md

```diff
@@ -23,6 +23,7 @@ scripts/start.js
 ```
 
 
+## ReadMe from Create-React-App
```

Comment:
There is no need to put "ReadMe"in the title. Right? :) Just add a title like "Demo of Front-End with React" ...